jet.el

```diff
@@ -71,10 +71,12 @@
 
 (defun jet--thing-at-point ()
   "Return the active region or the thing at point."
-  (string-trim
-   (if (use-region-p)
-       (buffer-substring-no-properties (region-beginning) (region-end))
-     (thing-at-point 'sexp t))))
+  (when-let ((sexp (or (thing-at-point 'sexp t)
+                       (thing-at-point 'defun t))))
```

Comment:
Shouldn't we do this only in the else of the if below?

Reply:
Hm I suppose so yeah